users/users-model.js

```diff
@@ -0,0 +1,32 @@
+const knex = require('knex');
+
+const dbConfig = require('../knexfile.js');
+
+const db = knex(dbConfig.development);
+
+module.exports = {
+  add,
+  find,
+  findBy,
+  findById,
+};
+
+function find() {
+  return db('users').select('id', 'username', 'password');
```

Comment:
Is the `select` needed when all columns of the table are being selected? You could also optionally just use a [wildcard](https://en.wikipedia.org/wiki/Wildcard_character) to select all tables:
```javascript
  return db('users').select('*');
```
but I say, why even have the extra step/method?
```javascript
  return db('users');
```

Reply:
I like it.  I also didn't know I could use a wildcard.  Thanks